package_manifest.yaml

```diff
@@ -453,6 +453,7 @@ foreman_proxy_packages:
     rubygem-rake-compiler: {}
     rubygem-rubyipmi: {}
     rubygem-rb-inotify: {}
+    rubygem-rkerberos: {}
```

Comment:
you add the package to `foreman_proxy_packages`, which obal thinks is built for non-SCL, but you're actually building an SCL'ed version, so the result fails repoclosure.